lua/hurl/hurl.lua

```diff
@@ -28,16 +41,23 @@ function M.hurl(config)
         vim.api.nvim_win_close(win_id, true)
       end
       -- Clean this autocmd up
-      vim.api.nvim_del_autocmd(args.id)
+      return true
     end,
   })
 
   -- Build arguments list
-  local hurl_args_t = {}
-  if config.color then
-    table.insert(hurl_args_t, "--color")
-  else
-    table.insert(hurl_args_t, "--no-color")
+  local hurl_args_t = vim.tbl_deep_extend("force", {}, config.hurl_flags)
+  -- Always ensure the `--include` flag is passed so we can get http headers
+  if not list_contains(hurl_args_t, "--include") then
```

Comment:
I'd rather not force the include flag. Most of the time I personally do not want to show headers. We could allow configuring a default list of options in the config though.

If we go the default options list we could clean up the color vs no-color thing as well by just using whatever is in the list instead. We could also move away from the ":Hurl" and ":HurlNoColor" and instead do something like ":Hurl" and ":HurlNoDefaults"

Reply:
I was jumping the gun a bit, later on to resolve #7 we are always going to need `--include` as part of the flags so we can extract filetype info. For now, though, you're right, this should be reincluded when that functionality is being implemented.

Will remove.